edx_user_state_client/tests.py

```diff
@@ -373,7 +386,63 @@ def test_set_many_with_history(self):
         )
 
 
-class UserStateClientTestBase(_UserStateClientTestCRUD, _UserStateClientTestHistory):
+class _UserStateClientTestIterAll(_UserStateClientTestUtils):
+    """
+    Blackbox tests of basic XBlockUserStateClient global iteration functionality.
+    """
+
+    __test__ = False
+
+    def test_iter_blocks_empty(self):
+        self.assertItemsEqual(
+            self.iter_all_for_block(0),
+            []
+        )
+
+    def test_iter_blocks_single_user(self):
+        self.set_many(0, {0: {'a': 'b'}, 1: {'c': 'd'}})
+
+        self.assertItemsEqual(
+            (item.state for item in self.iter_all_for_block(0)),
+            [{'a': 'b'}]
+        )
+
+        self.assertItemsEqual(
+            (item.state for item in self.iter_all_for_block(1)),
+            [{'c': 'd'}]
+        )
+
+    def test_iter_blocks_many_users(self):
+        for user in xrange(3):
+            self.set_many(user, {0: {'a': user}, 1: {'c': user}})
+
+        self.assertItemsEqual(
+            ((item.username, item.state) for item in self.iter_all_for_block(0)),
+            [
+                (self._user(0), {'a': 0}),
+                (self._user(1), {'a': 1}),
+                (self._user(2), {'a': 2}),
+            ]
+        )
+
+    def test_iter_blocks_deleted_block(self):
+        for user in xrange(3):
+            self.set_many(user, {0: {'a': user}, 1: {'c': user}})
+
+        self.delete(1, 0)
```

Comment:
A single comment here would help readability - what's the `1` and `0` parameters?